addons/hearing/functions/fnc_firedNear.sqf

```diff
@@ -88,8 +87,15 @@ _caliber = call {
     if (_caliber <= 0) then { 6.5 } else { _caliber };
 };
 _loudness = (_caliber ^ 1.25 / 10) * (_initspeed / 1000) * _audibleFireCoef / 5;
+
+_loudnessCoef = getNumber (_ammoConfig >> QGVAR(loudnessCoef));
```

Comment:
This approach has a big downside; if the loudnessCoef is not defined for a given weapon (e.g., third party) they will no longer produce hearing loss, as getNumber returns zero. 

An alternative would be to reverse the coefficient and call it `attenuationCoef`, so the default is zero (no attenuation) and the maximum is 1 (total attenuation).